cmd/crio/main.go

```diff
@@ -21,6 +20,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/soheilhy/cmux"
 	"github.com/urfave/cli"
+	"golang.org/x/net/context"
```

Comment:
Do we want this change?  [Their docs have][1]:

>  As of Go 1.7 this package is available in the standard library under the name context. https://golang.org/pkg/context. 

Maybe we have a dependency that needs updating to use the stdlib?

[1]: https://godoc.org/golang.org/x/net/context

Reply:
i'm two ways. there are so many places to update this. I updated most of them, but when changing this one, it fails compile due to signature mis-match with the context usage in a subpackage